device/tests/Microsoft.Azure.Devices.Client.Test/DeviceClientTests.cs

```diff
@@ -27,6 +27,35 @@ public void DeviceClient_OperationTimeoutInMilliseconds_Property_DefaultValue()
         }
 
         [TestMethod]
+        [TestCategory("IoTHubClientDiagnostic")]
+        public void DeviceClient_SetDiagnosticSamplingPercentage()
```

Comment:
Please separate the tests and name them such that they are documenting the functional requirements:
`DeviceClient_DefaultDiagnosticSamplingPercentage_Ok`
`DeviceClient_SetDiagnosticSamplingPercentageInRange_Ok`
`DeviceClient_SetDiagnosticSamplingPercentageOutOfRange_Fail`

Reply:
fixed